mantidproject/mantid
# Added missing tolerances for CompressEvents in AlignAndFocusPowderFromFiles

**Description of work.**

Added the tolerances `CompressTolerance` and `CompressWallClockTolerance` inputs from `AlignAndFocusPowderFromFiles` to the child algorithm calls to `CompressEvents` (int `AlignAndFocusPowderFromFiles`, not just the `AlignAndFocusPowder` algorithm) as the necessary `Tolerance` and `WallClockTolerance` arguments, respectively.

**To test:**

1) Set logging to `information` level.

2) Call `AlignAndFocusPowderFromFiles` **with at least two files to load** and `CompressTolerance` = 0. Will notice that `CompressEvents` is no longer called. 

3) Also, do with `CompressTolerance` > 0. and != 1e-5 and will see that `CompressEvents` called with the set value (not just the default).

4) Similarly,  test with the `CompressWallClockTolerance` argument set and see that its value is now set in `CompressEvents` as well.

Fixes *There is no associated issue.*

Release note added for v3.14.0

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
